Make.defaults

```diff
@@ -22,11 +22,15 @@ EFI_ARCHES ?= aa64 ia32 x64
 
 enabled = $(if $(filter undefined,$(origin $(1))),$(3),$(2))
 
+HOSTARCH   = $(shell uname -m | sed s,i[3456789]86,ia32,)
+ARCH	   := $(shell uname -m | sed s,i[3456789]86,ia32,)
+
+
 PKG_CONFIG ?= $(CROSS_COMPILE)pkg-config
 CC	:= $(if $(filter default,$(origin CC)),$(CROSS_COMPILE)gcc,$(CC))
 CCLD	:= $(if $(filter undefined,$(origin CCLD)),$(CC),$(CCLD))
 CFLAGS	?= -O2 -g3 -pipe -fPIE -fstack-protector-all \
-	-fstack-clash-protection -fcf-protection=full
+	-fstack-clash-protection $(ifneq(($ARCH), aa64),-fcf-protection=full,)
```

Comment:
Doesn't it only exist on amd64? This will allow it on anything but arm64, right?

Reply:
Race condition with your review and me pushing.  Will open another PR.